test/demo/realm.yml

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
This directory has been reassigned to be test_playbook.
I think, the examples could be in the inline documentation comment of the module.